lib/streamlit/runtime/state/query_params_proxy.py

```diff
@@ -68,15 +69,51 @@ def __setattr__(self, key: str, value: Union[str, Iterable[str]]) -> None:
 
     @gather_metrics("query_params.get_all")
     def get_all(self, key: str) -> List[str]:
+        """
+        Get a list of all query parameter values associated to a given key.
+
+        When a key is repeated as a query parameter within the URL, this method
+        allows all values to be obtained. In contrast, dict-like methods only
+        retireve tha last value when a key is repeated in the URL.
```

Comment:
typo:
```suggestion
        retrieve the last value when a key is repeated in the URL.
```

Reply:
Thanks. (I should have double-checked my Christmas Eve-Eve-Eve work.) 😅